.github/workflows/build-test.yml

```diff
@@ -32,23 +32,21 @@ jobs:
         working-directory: v2/
 
       - name: Build
-        run: go build .
-        working-directory: v2/cmd/nuclei/
+        run: bash build.sh retry 3 make build
+        working-directory: v2/
 
```

Comment:
@ShubhamRasal , not sure if this helpful . If I am not wrong  build failures only occur when there is something wrong with  `go.mod` or `go.sum`
Ex:
a dependecy requires min go version of 1.19 but go mod has 1.18
dependency tree errors which are caused due to importing old version of dep etc

I think retrying build will not be helpful since these errors are not random and retry won't affect it

Reply:
makes sense.. will remove retry from build